test/core/componentGroupTests.ts

```diff
@@ -65,7 +65,8 @@ describe("ComponentGroups", () => {
     componentGroup.remove(c1);
     assert.deepEqual(componentGroup.components(), [c0, c2],
     "removing a Component not in the Group does not remove Components from the Group");
-    assert.strictEqual(svg.node().childNodes[0], (<D3.Selection> (<any> c1)._element).node(), "The Component not in the Group stayed put");
+    assert.strictEqual((<Node> svg.node()).childNodes[0], (<d3.Selection<void>> (<any> c1)._element).node(),
```

Comment:
But it's not an `<any>`. You cast the `_element`, which is of type `d3.Selection<void>` anyway